### PR TITLE
Logged in users can change their password

### PR DESCRIPTION
**Pull request by katstevens**
Buyer, Supplier and Admin users should see a password change link on their dashboard.

Clicking the link goes to a Change Password form where the user must enter their old password, a new password and confirm the new password.

After the password has been successfully changed, the user is redirected back to their dashboard.

### CODE REVIEW

**Comment by CrystalPea** [comment]
We added steps for testing that password-change-alert email is being send for all roles, and so we need someone to look at the code again, as it has been approved before those changes. @samuelhwilliams do you think you could review these additional steps for us?

File: features/user/change_password.feature [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@password_change @notify @skip-staging
```
[the remark this answers is not in the document]
Comment (CrystalPea) [reply features/user/change_password.feature]: @samuelhwilliams good call, changed that now. We also added another scenario where validation is checked after consultation with @andrewchong

**Comment by CrystalPea** [comment]
@samuelhwilliams good call, changed that now. We also added another scenario where validation is checked after consultation with @andrewchong

**Comment by samuelhwilliams** [comment]
Looks good 👍 

screen shot 2018-05-01 at 14 40 16

**Pull request merged**